lib/logger.js

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright (c) 2016-2018 Digital Bazaar, Inc. All rights reserved.
+ */
+const bedrock = require('bedrock');
+
+module.exports = bedrock.loggers.get('app').child('bedrock-document-server');
```

Comment:
convention is not to include `bedrock` in the child logger name.

Reply:
Considering the purpose of namespaces is to add scope clarity, I'm not sure why a convention appeared that uses truncated names.  I guess just to reduce log line length?  Short names increase the possibility of conflicts.